ted/gtfs.py

```diff
@@ -127,62 +127,62 @@ def remove_routes_from_gtfs(gtfs_path: str, output_folder: str, route_ids: list[
         os.mkdir(output_folder)
     gtfs.write_zip(os.path.join(output_folder, zipfile_name))
 
-# def remove_premeium_routes_from_gtfs(gtfs_path: str, output_path: str, premium_routes_path: str):
-#     """Make a copy of a GTFS folder without premium routes
-
-#     Parameters
-#     ----------
-#     gtfs_path : str
-#         The path to the folder to remove premium routes from.
-#     output_path : str
-#         The path to where the new GTFS folder without premium routes will be created.
-#     premium_routes_path : str
-#         The path to the csv containing the list of premium route slugs and their ids.
-#         This must specify a csv file and the csv should be formatted into 'route_slug, route_id' columns
-
-#     """
-#     premium_routes = pandas.read_csv(premium_routes_path, index_col = False)
-#     os.mkdir(output_path)
-#     dated_entries = os.listdir(gtfs_path)
-
-#     # Iterate through all dated entries
-#     for curr_dated_entry in dated_entries:
-#         # Make target output folder with the '-limited' tag
-#         dated_output_path = os.path.join(output_path, curr_dated_entry + '-limited')
-#         os.mkdir(dated_output_path)
-#         zip_entries = os.listdir(os.path.join(gtfs_path, curr_dated_entry))
-#         # Iterate through .zip entries
-#         for curr_zip_entry in zip_entries:
-#             # Find entry zip folder
-#             curr_zip_dir = os.path.join(gtfs_path, curr_dated_entry, curr_zip_entry)
-#             curr_zip_slug = curr_zip_entry.removesuffix('.zip')
+def remove_premeium_routes_from_gtfs(gtfs_path: str, output_path: str, premium_routes_path: str):
+    """Make a copy of a GTFS folder without premium routes
+
+    Parameters
+    ----------
+    gtfs_path : str
+        The path to the folder to remove premium routes from.
+    output_path : str
+        The path to where the new GTFS folder without premium routes will be created.
+    premium_routes_path : str
+        The path to the csv containing the list of premium route slugs and their ids.
+        This must specify a csv file and the csv should be formatted into 'route_slug, route_id' columns
+
+    """
+    premium_routes = pandas.read_csv(premium_routes_path, index_col = False)
+    os.mkdir(output_path)
```

Comment:
Something you can do here is:

``` python
if not os.path.exists(output_folder):
    os.mkdir(output_folder)
```

That way if it already exists it won't throw an error.